commands/deploy.go

```diff
@@ -86,7 +85,7 @@ func (c *DeployCmd) Run() (err error) {
 		return fmt.Errorf("unable to seek to beginning of tarball: %s", err)
 	}
 
-	fmt.Printf("Pushing %d bytes...\n", stat.Size())
+	fmt.Printf("Pushing %dMB...\n", stat.Size()/1024)
```

Comment:
Is `Size()` returning kilobytes?

Reply:
I believe it's bytes. 

[godoc says](https://golang.org/pkg/os/#FileInfo):

```
Size() int64        // length in bytes for regular files; system-dependent for others
```

Reply:
Then you need to divide by `1024` twice in order to get `MB`. Your message is wrong.

Reply:
😂 I have never felt more post-technical in my life.